parmed/formats/pdb.py

```diff
@@ -722,7 +722,7 @@ def print_atoms(atom, coords):
                 else:
                     atoms = sorted(res.atoms, key=lambda atom: atom.number)
                 for atom in atoms:
-                    pa, others, (x, y, z) = print_atoms(atom, coords[0])
+                    pa, others, (x, y, z) = print_atoms(atom, coord)
```

Comment:
using 'print_atoms' seems not a good function name. :D I always think ``print`` is just for stdout.